example/hands-on/config/e2e_config.yml

```diff
@@ -0,0 +1,15 @@
+:selenium_ip: 'Your selenium ip'  # default selenium ip. please set ip or 'docker_host_ip'. ip route | awk 'NR==1 {print $3}'
```

Comment:
why do you commit this file?

Reply:
I commit all files which are created in this example.